src/backend/joanie/debug/views.py

```diff
@@ -79,6 +82,114 @@ class DebugMailSuccessPaymentViewTxt(DebugMailSuccessPayment):
     template_name = "mail/text/order_validated.txt"
 
 
+class DebugMailSuccessInstallmentPaid(TemplateView):
+    """Debug View to check the layout of the success installment payment by email"""
+
+    def get_context_data(self, **kwargs):
+        """
+        Base method to prepare the document context to render in the email for the debug view.
+        Usage reminder :
+            /__debug__/mail/installment_paid_html
+        """
+        product = ProductFactory(price=Decimal("1000.00"))
+        product.set_current_language("en-us")
+        product.title = "Test product"
+        product.set_current_language("fr-fr")
+        product.title = "Test produit"
+        product.save()
+        order = OrderGeneratorFactory(
+            product=product,
+            state=ORDER_STATE_PENDING_PAYMENT,
+            owner=UserFactory(first_name="John", last_name="Doe", language="en-us"),
+        )
+        invoice = Invoice.objects.create(
+            order=order,
+            parent=order.main_invoice,
+            total=0,
+            recipient_address=order.main_invoice.recipient_address,
+        )
+        for payment in order.payment_schedule[:2]:
+            payment["state"] = PAYMENT_STATE_PAID
+            Transaction.objects.create(
+                total=Decimal(payment["amount"].amount),
+                invoice=invoice,
+                reference=payment["id"],
+            )
+        current_language = translation.get_language()
+        with translation.override(current_language):
+            product.set_current_language(current_language)
+            return super().get_context_data(
+                order=order,
+                course_title=product.title,
+                order_payment_schedule=order.payment_schedule,
+                amount_debited=Money(order.payment_schedule[2]["amount"]),
+                complete_cost=Money(order.product.price),
+                installment_position_paid=order.get_count_installments_paid(),
+                balance_remaining_to_be_paid=order.get_remaining_balance_to_pay(),
+                next_installment_date=order.get_date_next_installment_to_pay(),
+                credit_card_last_four_numbers=order.credit_card.last_numbers,
+                installment_concerned_position=order.get_position_last_paid_installment(),
```

Comment:
IMHO, we should have variable names that matches called functions.
```suggestion
                product_title=product.title,
                order_payment_schedule=order.payment_schedule,
                installment_amount=Money(order.payment_schedule[2]["amount"]),
                product_price=Money(order.product.price),
                count_installments_paid=order.get_count_installments_paid(),
                remaining_balance_to_pay=order.get_remaining_balance_to_pay(),
                date_next_installment_to_pay=order.get_date_next_installment_to_pay(),
                credit_card_last_numbers=order.credit_card.last_numbers,
                position_last_paid_installment=order.get_position_last_paid_installment(),
```